### DISCUSSION TITLE
How can I access union name?

### DISCUSSION THREAD

**Discussion by johannbrynjar** [MichalLytek/type-graphql#1300] [Q&A] [answered]
Can I have access to the union name when I create a union type?
```typescript
const SearchResultUnion = createUnionType({
  name: "SearchResult", 
  types: () => [Movie, Actor] as const,
});
```
I am trying to create a generic `PaginatedResponseClass` with a union object type.

**Comment by MichalLytek** [comment, marked as answer]
I don't think it's easily possible. You need to pass the name explicitly to the generic function.
